callstats-java-sdk/src/main/java/io/callstats/sdk/ServerInfo.java

```diff
@@ -14,30 +11,12 @@
 	/**
 	 * Instantiates a new endpoint info.
 	 */
-	public EndpointInfo() {
+	public ServerInfo() {
```

Comment:
Actually I think the old name 'EndpointInfo' is fine. Somebody advocate to change it to 'SeverInfo'? Ref: https://en.wikipedia.org/wiki/H.323  "Collectively, terminals, multipoint control units and gateways are often referred to as endpoints." So in H323 world, it's a media server is just one type of endpoint.